docs/asciidoc/modules/avaje-validator.adoc

```diff
@@ -0,0 +1,274 @@
+== Avaje Validator
```

Comment:
we also need a reference to this file here https://github.com/jooby-project/jooby/blob/3.x/docs/asciidoc/modules/modules.adoc#validation